src/participant_network.star

```diff
@@ -88,7 +91,16 @@ def launch_participant_network(plan, participants, network_params, global_log_le
 			fail("Unsupported launcher '{0}', need one of '{1}'".format(el_client_type, ",".join([el.name for el in el_launchers.keys()])))
 
 		el_launcher, launch_method = el_launchers[el_client_type]["launcher"], el_launchers[el_client_type]["launch_method"]
-		el_service_name = "el-{0}-{1}-{2}".format(index+1, el_client_type, cl_client_type)
+
+		for par, zf in zip(package_io.GLOBAL_INDEX_ZFILL['participants'], package_io.GLOBAL_INDEX_ZFILL['zfill']):
```

Comment:
can lines 95-101 be pulled into a util that gets shared with lines  170-176, also can the zfil be different? isn't num el == num cl? we are just calculating the same thing again

Reply:
moved it to its own def